### PR TITLE
Update fuzziness in css3/ for accelerated drawing

### PR DESCRIPTION
**Pull request by nmoucht**
#### 25faabfd88236fe3164c200ac214cc546f56172b
<pre>
Update fuzziness in css3/ for accelerated drawing
<a href="https://bugs.webkit.org/show_bug.cgi?id=246832">https://bugs.webkit.org/show_bug.cgi?id=246832</a>
rdar://101408931

Reviewed by Simon Fraser.

Update fuzziness in css3/ for when accelerated drawing is enabled
by default.

* LayoutTests/css3/background/background-repeat-space-content.html:
* LayoutTests/css3/background/background-repeat-space-padding.html:
* LayoutTests/css3/blending/background-blend-mode-body-image.html:
* LayoutTests/css3/blending/background-blend-mode-data-uri-svg-image.html:
* LayoutTests/css3/color-filters/color-filter-color-text-decorations.html:
* LayoutTests/css3/color-filters/color-filter-text-decoration-shadow.html:
* LayoutTests/css3/masking/mask-base64.html:
* LayoutTests/css3/masking/mask-luminance-gradient.html:
* LayoutTests/css3/masking/mask-repeat-round-content.html:
* LayoutTests/css3/masking/mask-repeat-space-border.html:

Canonical link: <a href="https://commits.webkit.org/255977@main">https://commits.webkit.org/255977@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/4c5c0c64e4548f5352d198093146bbfd8a142c06

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/94222 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/3414 "Built successfully") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/25023 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/103904 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/164180 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/3441 "Built successfully") | [  ~~🛠 mac-debug~~](https://ews-build.webkit.org/#/builders/71/builds/31628 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/86578 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/99925 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/99878 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/2466 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/80634 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/86578 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/84385 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/25023 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/86578 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/66/builds/38010 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/25023 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/35894 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/25023 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/39772 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/80634 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1945 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/41717 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/25023 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | |

### CODE REVIEW

**Review by smfr — APPROVED** [review]

File: Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def parse_args(args):
     option_group_definitions.append(("Feature Switches", [
         optparse.make_option("--complex-text", action="store_true", default=False,
             help="Use the complex text code path for all text (OS X and Windows only)"),
-        optparse.make_option("--accelerated-drawing", action="store_true", default=False,
+        optparse.make_option("--accelerated-drawing", action="store_true", default=True,
```
Comment (JonWBedard) [inline Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py]: This feels like it should be a separate PR? And conflicts with #5523, right?
